src/libinjector/libinjector.h

```diff
@@ -105,6 +105,27 @@
 #ifndef LIBINJECTOR_H
 #define LIBINJECTOR_H
 
+#include "../print_util.h"
+
+inline void print_injection_info(output_format_t format, vmi_pid_t pid, const char* file)
```

Comment:
Looking at this again I wonder why not just print this information from `injector_start_app`? There it could also print extra information that is useful, like what is the PID/dtb of the new process.

Reply:
It is called from two places: 1) from main.cpp of drakvuf and 2) from injector.c of injector.
It seems like injector does not need this info printed.

Reply:
I think it would be useful to print in both cases, from injector it would just always print in the default format, not in csv.

Reply:
There is one more problem. No format is available in this function. 
I need the print it in a special format not csv and not default. 
I'll have to pass format inside this function. 
Is it Ok?

Reply:
It easily could. Just asking.